lib/utils/createPinrowSilkscreenText.ts

```diff
@@ -0,0 +1,27 @@
+import type { PrimitiveComponent } from "lib/components/base-components/PrimitiveComponent"
+import { SilkscreenText } from "lib/components/primitive-components/SilkscreenText"
+
+export const createPinrowSilkscreenText = ({
+  elm,
+  pinLabels,
+  readableRotation,
+}: { elm: any; pinLabels: any; readableRotation: number | undefined }) => {
```

Comment:
```suggestion
}: { elm: any; pinLabels: any; readableRotation: number }) => {
```

Why `undefined` we can pass some default value to the function itself?

Reply:
Same for the other parameters in this function signature should have defined types